coordinator/src/services/translation/request/dispense/release.ts

```diff
@@ -0,0 +1,60 @@
+import * as hl7v3 from "../../../../models/hl7-v3"
+import * as uuid from "uuid"
+import * as fhir from "../../../../models/fhir"
+import {getIdentifierParameterByName} from "../../common"
+
+export function translateReleaseRequest(
+  fhirReleaseRequest: fhir.Parameters
+): hl7v3.NominatedPrescriptionReleaseRequestWrapper {
+  const hl7Id = new hl7v3.GlobalIdentifier(uuid.v4())
+  const timestamp = new hl7v3.Timestamp("")
```

Comment:
Could use the current time for this instead of a blank string:
```
convertMomentToHl7V3DateTime(moment.utc())
```
In case Spine cares about it being populated